components/infobox/wikis/deadlock/infobox_item_custom.lua

```diff
@@ -0,0 +1,231 @@
+---
+-- @Liquipedia
+-- wiki=deadlock
+-- page=Module:Infobox/Item/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Logic = require('Module:Logic')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector')
+local Item = Lua.import('Module:Infobox/Item')
+
+local AutoInlineIcon = Lua.import('Module:AutoInlineIcon')
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+local Center = Widgets.Center
+
+---@class DeadlockItemInfobox: ItemInfobox
+local CustomItem = Class.new(Item)
+local CustomInjector = Class.new(Injector)
+
+local CATEGORY_DISPLAY = {
+	['weapon'] = 'Weapon [[Category:Weapon Items]]',
+	['vitality'] = 'Vitality [[Category:Vitality Items]]',
+	['spirit'] = 'Spirit [[Category:Spirit Items]]',
+}
```

Comment:
Split the display from the category setting

Reply:
I'm a bit lost on what you're asking / what this does and what Im supposed to turn it into

Reply:
`[[Category:Spirit Items]]` is a lazy way of adding `Spirit Items` as a category to the page. 
Instead add the categories in the `getWikiCategories` function

Reply:
```suggestion
local CATEGORIES = {
	weapon = 'Weapon',
	vitality = 'Vitality',
	spirit = 'Spirit',
}
```